### PR TITLE
Tutorial added on 1417 - Forwarding Emails (en, bn)

### PR DESCRIPTION
**Pull request by subhashiscse**

### CODE REVIEW

File: 1417/en.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+If you want to send mail to 1st person to 3rd person, you have to send it through 2nd person.
+Now in this problem you have to find the first person so that, you may send mail to maximum number of people.
+We may try to understand the problem through an example.
+
+Example:
+1 2 - 1st person can send mail to 2nd person
+2 1 - 2nd person can send mail to 1st person
+5 3 - 5th person can send mail to 3rd person
+3 4 - 3rd person can send mail to 4th person
+4 5 - 4th person can send mail to 5th person
```
Comment (rebornplusplus) [inline 1417/en.md:16]: * Try including a drawing of the graph. https://csacademy.com/app/graph_editor/
* Markdown makes these 5 lines into 1, resulting in poor readability. Use `<br/>` I guess.

File: 1417/bn.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+এখন এই প্রবলেমে তোমাকে বলে দেওয়া থাকবে, কার সাথে কার যোগাযোগ আছে অর্থাৎ কে কাকে মেইল পাঠাতে পারবে।
+তোমাকে বলতে হবে কাকে প্রথমে মেইল পাঠালে তুমি বেশি সংখ্যক মানুষের কাছে মেইল পৌঁছাতে পারবা?
+আমরা একটা ছোট্ট উদাহরণ এর মাধ্যমে জিনিসটা বোঝার চেষ্টা করি। 
+
+উদাহরণঃ 
+১ ২ - ১ম জন ২য় জনকে মেইল পাঠাতে পারবে 
+২ ১ - ২য় জন ১ম জনকে মেইল পাঠাতে পারবে
+৫ ৩ - ৫ম জন ৩য় জনকে মেইল পাঠাতে পারবে
+৩ ৪ - ৩য় জন ৪র্থ জনকে মেইল পাঠাতে পারবে
+৪ ৫ - ৪র্থ জন ৫ম জনকে মেইল পাঠাতে পারবে
```
Comment (rebornplusplus) [inline 1417/bn.md:17]: * Try including a drawing of the graph. https://csacademy.com/app/graph_editor/
* Markdown makes these 5 lines into 1, resulting in poor readability. Use `<br/>` I guess.

**Review by rebornplusplus — CHANGES_REQUESTED** [review]

**Comment by subhashiscse** [comment]
I add the graph image.Can you please check again?

**Comment by rebornplusplus** [comment]
* You forgot to link the image in `en.md`.
* Maybe you should crop the image a little bit. It's quite big. (But you can ignore this.)

**Comment by subhashiscse** [comment]
Yeah my mistake to add the image link in en.md.
Now add it and image crop

**Review by rebornplusplus — APPROVED** [review]
Everything looks okay. Good work!

**Pull request merged**